### PR TITLE
ps1 build without parameters

### PR DESCRIPTION
**Pull request by markuskramerIgitt**
This PR contains mainly
- The build script in Powershell and without parameters.
- Note on Productversion

### CODE REVIEW

**Comment by markuskramerIgitt** [comment]
@twangboy please review

File: wix.d/MinionMSI/WixUI_Minion.wxs [hunk 1]
```diff
@@ -19,12 +19,12 @@
           SettingsCustomizationDlg == Hostname Specification
           VerifyReadyDlg           == Ready to install
        -->
-      <Publish Dialog="InstallDirDlg" Control="Next" Event="NewDialog" Value="SettingsCustomizationDlg" Order="4">WIXUI_DONTVALIDATEPATH OR WIXUI_INSTALLDIR_VALID="1"</Publish>
+      <Publish Dialog="InstallDirDlg"            Control="Next" Event="NewDialog" Value="SettingsCustomizationDlg" Order="4">WIXUI_DONTVALIDATEPATH OR WIXUI_INSTALLDIR_VALID="1"</Publish>
```
Comment (twangboy) [inline wix.d/MinionMSI/WixUI_Minion.wxs]: Is there a reason for these additional spaces?

File: wix.d/MinionMSI/WixUI_Minion.wxs [hunk 2]
```diff
@@ -19,12 +19,12 @@
...
 
       <Publish Dialog="SettingsCustomizationDlg" Control="Next" Event="NewDialog" Value="VerifyReadyDlg">1</Publish>
       <Publish Dialog="SettingsCustomizationDlg" Control="Back" Event="NewDialog" Value="InstallDirDlg">1</Publish>
 
-      <Publish Dialog="VerifyReadyDlg" Control="Back" Event="NewDialog" Value="SettingsCustomizationDlg" Order="1">NOT Installed</Publish>
+      <Publish Dialog="VerifyReadyDlg"           Control="Back" Event="NewDialog" Value="SettingsCustomizationDlg" Order="1">NOT Installed</Publish>
```
Comment (twangboy) [inline wix.d/MinionMSI/WixUI_Minion.wxs]: Same here

File: wix.d/MinionMSI/WixUI_Minion.wxs (hunk 1 above)
Comment (markuskramerIgitt) [reply wix.d/MinionMSI/WixUI_Minion.wxs]: You can increase readability of XML tags written below  when you vertically align their attributes.

In this case, 4 `Publish` tags in ProductUI.wxs have the attribute `Control`, so I aligned 2 of them.

**Pull request merged**